tsl/test/expected/chunk_api-12.out

```diff
@@ -85,6 +94,58 @@ SELECT * FROM _timescaledb_internal.create_chunk('chunkapi',' {"time": [15150240
+-- Missing dimension
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": [1514419600000000, 1515024000000000]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid hypercube for hypertable "chunkapi"
+DETAIL:  invalid number of hypercube dimensions
+-- Extra dimension
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": [1514419600000000, 1515024000000000],  "device": [-9223372036854775808, 1073741823], "time2": [1514419600000000, 1515024000000000]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid hypercube for hypertable "chunkapi"
+DETAIL:  invalid number of hypercube dimensions
+-- Bad dimension name
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": [1514419600000000, 1515024000000000],  "dev": [-9223372036854775808, 1073741823]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid hypercube for hypertable "chunkapi"
+DETAIL:  dimension "dev" does not exist in hypertable
+-- Same dimension twice
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": [1514419600000000, 1515024000000000], "time": [1514419600000000, 1515024000000000]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid hypercube for hypertable "chunkapi"
+DETAIL:  invalid number of hypercube dimensions
+-- Bad bounds format
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": ["1514419200000000", 1515024000000000], "device": [-9223372036854775808, 1073741823]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid hypercube for hypertable "chunkapi"
+DETAIL:  constraint for dimension "time" is not numeric
+-- Bad slices format
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": [1515024000000000], "device": [-9223372036854775808, 1073741823]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid hypercube for hypertable "chunkapi"
+DETAIL:  unexpected number of dimensional bounds for dimension "time"
+-- Bad slices json
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time: [1515024000000000] "device": [-9223372036854775808, 1073741823]}', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  invalid input syntax for type json
+LINE 1: ...imescaledb_internal.create_chunk_table('chunkapi',' {"time: ...
+                                                             ^
+DETAIL:  Token "device" is invalid.
+CONTEXT:  JSON data, line 1:  {"time: [1515024000000000] "device...
+-- Valid chunk, but no permissions
+SET ROLE :ROLE_DEFAULT_PERM_USER_2;
+SELECT * FROM _timescaledb_internal.create_chunk_table('chunkapi',' {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]}', 'ChunkSchema', 'My_chunk_Table_name', '_timescaledb_internal','_hyper_1_1_chunk');
+ERROR:  function _timescaledb_internal.create_chunk_table(unknown, unknown, unknown, unknown, unknown, unknown) does not exist
```

Comment:
Is this expected output? Doesn't seem to match comment above.

Reply:
Thank you for noticing this! Fixed.